storages/backends/apache_libcloud.py

```diff
@@ -120,9 +122,16 @@ def size(self, name):
         else:
             return -1
 
-    def url(self, name):
+    def url(self, name, temporary_url=False, temporary_url_timeout=None):
```

Comment:
can we change these parameters to `temporary` and `timeout`? we're already part of the `url` method.